### ISSUE TITLE
Release version 1.0.3 to maven

### ISSUE THREAD

**Issue by tohyf** [nyholku/purejavacomm#135]
I noticed that the maven repo has only 1.0.2 version while this repo already had 1.0.3. Any plans to release version 1.0.3 to maven?

**Comment by tohyf** [comment]
is this maven yours? 
https://mvnrepository.com/artifact/org.bidib.com.github.purejavacomm/purejavacomm/1.0.3.2

**Comment by nyholku** [comment]
I don't think so, I have not made a Maven release for a couple of years

**Comment by WickedShell** [comment]
@nyholku any change of getting 1.0.3 pushed to maven?

**Comment by Chris2011** [comment]
Yeah, would be good to update the maven version too. In your releases you said you just renamed 1.0.2 to 1.0.3 to be in sync with everything. So maven needs this too. Not to be confused. As I understand it correctly, the maven artifact 1.0.2 is also 1.0.3 right?

**Comment by WickedShell** [comment]
@Chris2011 almost certainly not. The release on maven still says 2017, and I had to locally install 1.0.3 to upgrade my JNA version, which was needed to fix being completely unable to use serial ports on some Windows 10 machines.

**Comment by Chris2011** [comment]
Thx, good to know :)

**Comment by akuhtz** [comment]
> 
> 
> is this maven yours?
> https://mvnrepository.com/artifact/org.bidib.com.github.purejavacomm/purejavacomm/1.0.3.2

Hello,
This release comes from my forked repo based on branch bidib: https://github.com/akuhtz/purejavacomm/commits/bidib

**Comment by Chris2011** [comment]
@akuhtz so you fixed some stuff, right?

**Comment by akuhtz** [comment]
@Chris2011 Yes. My main goal was publish to central maven repo, but I merged some fixes, too. I just updated the readme and the changelog.
